src/mod2c_core/nocpout.c

```diff
@@ -1063,6 +1030,9 @@ static const char *_mechanism[] = {\n\
 		Lappendstr(defs_list, "extern Node* nrn_alloc_node_;\n");
 	}
 #endif
+    // TODO: these variables aldo declared as global (e.g. _ca_type in case of CaDynamics_E2.mod
+    // and used for nrn_wrote_conc function inside initmodel(). But I didn't see typical warning
+    // about usage of static variable?
```

Comment:
this needs to be tested by compiling mod file: https://bbpgitlab.epfl.ch/hpc/sim/models/neocortex/-/blob/main/mod/v5/CaDynamics_E2.mod
Should be covered by channel-benchmark: https://bbpgitlab.epfl.ch/hpc/channel-benchmark/-/tree/master/benchmark/channels/lib/modlib